channeld/channeld.c

```diff
@@ -588,7 +656,8 @@ static void channel_announcement_negotiate(struct peer *peer)
 	/* If we've completed the signature exchange, we can send a real
 	 * announcement, otherwise we send a temporary one */
 	if (peer->have_sigs[LOCAL] && peer->have_sigs[REMOTE]) {
-		check_short_ids_match(peer);
+		// check_short_ids_match(peer);
+		(void)check_short_ids_match;
```

Comment:
Ah that's a great find! Needed to update the code to update the short channel id only upon *mutual* splice lock otherwise this check would trigger when the peers saw blocks at slightly different times.